crates/bcr-wallet-core/src/lib.rs

```diff
@@ -307,6 +304,59 @@ pub async fn wallet_clean_local_db(idx: usize) -> u32 {
 
 // --------------------------------------------------------------- wallet_load_tx
 #[wasm_bindgen]
+#[derive(Clone, Copy, Default)]
+pub enum TransactionDirection {
+    #[default]
+    Incoming,
+    Outgoing,
+}
+impl std::convert::From<cdk::wallet::types::TransactionDirection> for TransactionDirection {
+    fn from(dir: cdk::wallet::types::TransactionDirection) -> Self {
+        match dir {
+            cdk::wallet::types::TransactionDirection::Incoming => TransactionDirection::Incoming,
+            cdk::wallet::types::TransactionDirection::Outgoing => TransactionDirection::Outgoing,
+        }
+    }
+}
+#[wasm_bindgen]
+#[derive(Clone, Copy, Default)]
+pub enum PaymentType {
+    #[default]
+    Na,
```

Comment:
Does this stand for something like "not available"? If so, I didn't find it very intuitive - maybe "Unknown", or "Empty", or "None"?